.circleci/config.yml

```diff
@@ -64,6 +64,24 @@ commands:
   non-coverage-tests:
     description: Do tests without tracking code coverage
     steps:
+      # Pre-build Sematic so we can retry if there are errors due to downloading deps
+      - run:
+          name: Pre-build Sematic
+          command: bazel build --experimental_repository_downloader_retries=5 //sematic/...
+      - run:
+          # Unfortunately this is the best way to retry in Circle CI
```

Comment:
I was thinking of using a script to relaunch a process if the exit code is not 0. It might be a bit more universal. similar to https://www.meziantou.net/retry-a-bash-command.htm

No intention to block this PR. But what do you think about that idea?

Reply:
I don't have a strong preference. This has the nice property that it's clear what's going on without looking at another file.